holidays/countries/bangladesh.py

```
@@ -50,16 +85,117 @@ def _populate_public_holidays(self):
+        2025: (MAR, 31),
+    }
+
+    # https://www.timeanddate.com/holidays/bangladesh/jumatul-bidah
+    JUMATUL_WIDA_DATES_CONFIRMED_YEARS = (2022, 2026)
+    JUMATUL_WIDA_DATES = {
+        2022: (APR, 29),
+        2023: (APR, 21),
+        2024: (APR, 5),
+        2025: (),
+        2026: (MAR, 20),
+    }
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)